pages/index.js

```diff
@@ -1,123 +1,51 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import { Inter } from '@next/font/google'
-import styles from '../styles/Home.module.css'
+import Head from "next/head";
+import Image from "next/image";
+import DisplayBasicInfo from "../components/BasicInfo";
+import DisplayEventInfo from "../components/Events/EventInfo";
+import NewPlayerPage from "../components/NewPlayerPage";
+import { Inter } from "@next/font/google";
+import styles from "../styles/Home.module.css";
 
-const inter = Inter({ subsets: ['latin'] })
+import data from "../BasicData.json";
+const inter = Inter({ subsets: ["latin"] });
 
-export default function Home() {
+import clientPromise from "../lib/mongodb";
+
+export async function getServerSideProps() {
```

Comment:
I think it's much better if we move this to `DisplayEventInfo` component. Because currently whole website can go down when this fails. If we moved, only `DisplayEventInfo` component will be broken.

Reply:
aha. ok. 👍 I see.

Reply:
I think whole website won't go down. it would be good to double check, though. You can throw any error in `try` phrase. `throw new Error('test')`